clients/drcachesim/tests/memref_gen.h

```diff
@@ -137,6 +145,41 @@ gen_exit(memref_tid_t tid)
     return memref;
 }
 
-} // namespace
+/* Returns a vector of encoded memref_t.
+ * The caller has to set tid + pid fields of the memref_t in memref_instr_t
```

Comment:
Done.

Reply:
Instead of adding the new sentence at the end saying this applies only to instr memrefs, I'd prefer modifying the original sentence so that it is clear from the start what it applies to.